do_it_yourself_notebook.ipynb

```diff
@@ -53,8 +53,8 @@
    "outputs": [],
    "source": [
     "# For simplicity, store the centre X and Y coords\n",
-    "centre_x = coordinates[1]\n",
-    "centre_y = coordinates[0]\n",
+    "centre_x = coordinates[1] #longitude\n",
```

Comment:
These should have a double space, according to the style guide! See: https://stackoverflow.com/a/1517774/2002176